packages/firebase/rollup-internal.config.js

```diff
@@ -19,18 +19,22 @@
  * Config for internal deployment, adds required license header to generated code.
  */
 
-import baseBuilds from './rollup.config.js';
+// This will only be used in a script that first renames rollup.config.js to rollup-main.config.js
+import baseBuilds from './rollup-main.config.js';
```

Comment:
Can you please add some comment for what this file is for? Something like 'It is only used in google3 to build bundles with license header '.

Does rollup not keep the license in the bundle ?

Reply:
Added more comments.  Rollup (or more precisely, the uglify plugin) doesn't automatically preserve the license in generated files unless you add a setting in config, which seems to be the same in webpack.